### ISSUE TITLE
Compiling on Ubuntu

### ISSUE THREAD

**Issue by scarletvv** [GeertLitjens/ASAP#28]
Hi,

I'm compiling ASAP on Ubuntu 14.04. May I ask when specifying the third party lib locations in CMake, should the variables point to *.a or *.so files? Thank you very much.

**Comment by GeertLitjens** [comment]
Hi,

They should normally point to the .so files, the .a files are statically linked, generally you want the dynamically linked .so files.

**Comment by scarletvv** [comment]
Hi,

Thank you for your reply. I'm getting another problem when compiling ASAP on Ubuntu 14.04. I compiled opencv 3.1 by myself. I have libopencv_world.so, libopencv_world.so.3.1, and libopencv_world.so.3.1.0 files inside the lib folder, but when compiling ASAP, I got the following error: 

*** No rule to make target `/home/cumed/Downloads/opencv/opencv3.1.0/build/lib/opencv_world310.lib', needed by `imgproc/NucleiSegmentation/libNucleiSegmentation.so'.  Stop.

I tried copying opencv_world310.lib to the lib folder of opencv and it solved the problem, but I'm just confused why opencv_world310.lib is needed when compiling on Ubuntu. Aren't .lib files for Windows system?
